types/aws-lambda/trigger/sqs.d.ts

```diff
@@ -44,3 +44,12 @@ export interface SQSMessageAttribute {
 export interface SQSMessageAttributes {
     [name: string]: SQSMessageAttribute;
 }
+
+// https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting
+export interface SQSEventResponse {
+    batchItemFailures: SQSBatchItemFailures[];
+}
+
+export interface SQSBatchItemFailures {
```

Comment:
This interface describes only _one_ failure
```suggestion
export interface SQSBatchItemFailure {
```